config/ember-try.js

```diff
@@ -1,62 +1,68 @@
-'use strict';
+"use strict";
 
-const getChannelURL = require('ember-source-channel-url');
+const getChannelURL = require("ember-source-channel-url");
+const typeTests = require("./ember-try-typescript");
 
-module.exports = async function() {
+const typeScriptScenarios = typeTests.scenarios.map((s) => ({
+  ...s,
+  command: typeTests.command,
+}));
```

Comment:
I'm not *in love* with this but it works while allowing the test config itself to remain simple *and* letting users do `ember try:each --config-path=./config/ember-try-typescript.json` to run *just* the type tests locally.

Reply:
this seems fine (especially while we iterate)